### ISSUE TITLE
Mario Tennis Ultra Smash: garbage name chars offline

### ISSUE THREAD

**Issue by Hexaae** [cemu-project/Cemu#900]
CEMU 2.0-44
Minor issue, but would be appreciated for offline gaming a fix on the Player name showing garbage chars ATM:
![Immagine 2023-07-14 163546](https://github.com/cemu-project/Cemu/assets/8591462/461236c3-c31d-4731-bcac-8bbd0031b23c)

Inject a an empty string or placeholder could be an acceptable solution.

**Comment by Squall-Leonhart** [comment]
It's chinese and says ?Panyan??Center

**Comment by goeiecool9999** [comment]
> It's chinese and says ?Panyan??Center

It's likely just interpreting random data as text. Probably means the game is getting an invalid pointer.

**Comment by Hexaae** [comment]
> > It's chinese and says ?Panyan??Center
> 
> It's likely just interpreting random data as text. Probably means the game is getting an invalid pointer.
Exactly. Since player _should be online_, the game from Yuzu offline seem to display random data as text from an invalid pointer. I think user defined account/profile name (Yuzu's General > System > Profile) should be injected in this case, even if offline.
